ts/module.ts

```diff
@@ -296,6 +296,10 @@ export class QuickJSWASMModule {
       runtime.setInterruptHandler(options.interruptHandler)
     }
 
+    if (options.maxStackSize) {
+      runtime.setMaxStackSize(options.maxStackSize)
```

Comment:
We also need a similar change in `module-asyncify.ts`. It's annoying to have a forked version of a function just for type shenanigans, maybe we should introduce a helper like `applyRuntimeOptions(runtime, options)`